org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/CompletionEngineProviderTests.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.jdt.core.tests.model;
+
+import junit.framework.Test;
+import org.eclipse.jdt.core.ICompilationUnit;
+import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.internal.compiler.impl.CompilerOptions;
+
+/**
+ * Test for the `org.eclipse.jdt.core.completionEngineProvider` extension point
+ * and {@link org.eclipse.jdt.internal.core.CompletionEngineProviderDiscovery}.
+ */
+public class CompletionEngineProviderTests extends AbstractJavaModelCompletionTests {
```

Comment:
Is it worth adding a `@SuppressWarnings("javadoc")` here to silence the warning, and just avoid increasing them ?